src/components/common/buttons/CustomButton.js

```diff
@@ -1,11 +1,10 @@
 // @flow
-import React from 'react'
+import React, { useCallback } from 'react'
 import { View } from 'react-native'
 import { ActivityIndicator, DefaultTheme } from 'react-native-paper'
 import { withStyles } from '../../../lib/styles'
 import Icon from '../view/Icon'
 import Text from '../view/Text'
-import useOnPress from '../../../lib/hooks/useOnPress'
```

Comment:
@MathiasMoralesDixtra if you fixed useOnPress why is it removed here?
@serdiukov-o-nordwhale can you jump in on this useOnPress issue

Reply:
For a strange reason, the useOnPress set the mnemonic value like undefined.

Reply:
I tested and I saw that adding the callback to the dependency array fix the issue. We can do useOnPress`(onPress,[onPress])` or to fix on all sites change this:
`const useOnPressHook = (callback, deps = [callback], options = {}) => `